docs/source/user-guide/sql/ddl.md

```diff
@@ -79,7 +79,7 @@ LOCATION <literal>
 
 `file_type` is one of `CSV`, `PARQUET`, `AVRO` or `JSON`
```

Comment:
I think we need to add `Arrow`, and add a example in doc

Reply:
Proposed change in https://github.com/apache/arrow-datafusion/pull/7489